packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/BarChartWidget.tsx

```diff
@@ -11,19 +11,22 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 import { BarChart, BarChartRepresentation } from '@eclipse-sirius/sirius-components-charts';
+import { GQLBarChart } from '@eclipse-sirius/sirius-components-forms';
 import { makeStyles } from '@material-ui/core/styles';
 import Typography from '@material-ui/core/Typography';
 import { useEffect, useRef, useState } from 'react';
-import { WidgetProps } from './WidgetEntry.types';
+import { BarChartWidgetProps } from './WidgetEntry.types';
 
 const useStyles = makeStyles((theme) => ({
   selected: {
     color: theme.palette.primary.main,
   },
 }));
 
-export const BarChartWidget = ({ widget, selection }: WidgetProps) => {
+export const BarChartWidget = ({ widget, selection }: BarChartWidgetProps) => {
```

Comment:
Perhaps a known limitation, but the "underline" and "strikethrough" flags do not seem to have any effect.

Reply:
Yes I don't know why "underline" and "strikethrough" flags  don't have any effect.